Homework-3/Homework-3/ViewController.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ViewController.swift
+//  Homework-3
+//
+//  Created by Kirill Skrytskii on 5.03.22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+        
+        sum(2.3, to: 1.6)
+        sum(6, to: 2, to: 3)
+        sum(30.2, by: 2)
+        let totalSumTupes = sumTupes(23.2, to: 43)
+        print(totalSumTupes)
+        
+        let sumDigits = digitSum(to: 2022)
+        print(sumDigits)
+        
+        string("Hello", to: "Hello")
+        print(power())
+        
+        let factorial = recursionFactorial(6)
+        print(factorial)
+    }
+    
+    // 1
+    
+    func sum(_ firstNumber: Double, to secondNumber: Double) {
+     print(firstNumber * secondNumber)
+        
+    }
+    
+    func sum(_ firsNumber: Int,to secondNumber: Int,to thirdNumber: Int ) {
+        print(firsNumber + secondNumber - thirdNumber)
+        
+    }
+    
+    func sum(_ firstNumber: Float, by secondNumber: Float) {
+        print(firstNumber / secondNumber)
+
+    }
+
+    func sumTupes(_ firstNumber: Double, to secondNumber: Int) -> Int {
+        return Int(firstNumber) + secondNumber
+    }
+
+// 2
+    
+    func digitSum(to number: Int) -> Int{
```

Comment:
I propose to rename your function to `digitsSum(of number: Int) -> Int`. Also, please, add a space before `{`. We always need spaces around `{` if it's not a new line

Reply:
what is the difference between (of , by , to ) in construction ?

Reply:
We should use that propositions that would be used in English sentence. For example, here I would say that we're calculation a digits sum of a number. That's why I propose to call the function `digitsSum(of number: Int) -> Int`